airflow/providers/mysql/hooks/mysql.py

```diff
@@ -16,10 +16,13 @@
 # specific language governing permissions and limitations
 # under the License.
 
+"""
+This module allows to connect to a MySQL database.
+"""
+
 import json
 
 import MySQLdb
```

Comment:
Do we need this import here? Maybe we can load this library only in specific cases?

Reply:
Not necessarily. I can move it to the `mysqlclient` code (so it will only be imported when used).